WordPress/Classes/ViewRelated/NUX/LoginSocialErrorViewController.swift

```diff
@@ -0,0 +1,153 @@
+import Foundation
+import Gridicons
+
+class LoginSocialErrorViewController: UITableViewController {
```

Comment:
This might have been a good candidate for [ImmuTable](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Utility/ImmuTable.swift).  Just FYI. No need to refactor.

Reply:
Ah, right. Hadn't even crossed my mind.